src/main/java/Player.java

```diff
@@ -0,0 +1,56 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Player {
+    private int settlements;
+    private int cities;
+    private int roads;
+    private int score;
+    private List<DevelopmentCard> developmentCards;
+    private Wallet wallet;
+    int numRoads;
+
+    public Player()
+    {
+        wallet = new Wallet();
+        developmentCards = new ArrayList<>();
+        cities = 4;
+        settlements = 5;
+        roads = 15;
+        score = 0;
+        numRoads = 0;
+    }
+
+    public void buildRoad(){
+        roads--;
```

Comment:
maybe only decrement after the player used cards so that if there aren't enough cards the roads isn't already decremented

Reply:
true, I'll add to check if there is any left first